### DISCUSSION TITLE
🧩 FEATURE REQUEST: short gif of the video content ad image thumbnail

### DISCUSSION THREAD

**Discussion by kimono76** [dailydotdev/daily#667] [Feature Requests]
### Description

When the post has at least one YouTube video, the image in the thumbnail should be a short gif of the first video
### Solution

Use the same solution that YouTube does when you stay in some video thumbnail

**Comment by idoshamun** [comment]
Hi @kimono76,

Thanks for the suggestion! We are right now focused still on written content but when we get to videos, we will definitely consider it
